docs/content/en/docs/Getting started/advanced-install.md

```diff
@@ -80,79 +80,79 @@ rabbitmq:
 The following table lists the configurable parameters of the chart and the default values.
 
 
-| Parameter                                                                   | Description                                                                                                        | Default                         |
```

Comment:
Was changing this intentional?

Reply:
The current table was not format correctly.
Before:
![Screenshot 2023-08-02 at 8 55 13 AM](https://github.com/m9sweeper/m9sweeper/assets/56838011/3a461671-f15b-4adb-a0dc-5a77ab969a02)
![Screenshot 2023-08-02 at 8 54 10 AM](https://github.com/m9sweeper/m9sweeper/assets/56838011/6cf3f9e8-8ab8-459b-b51b-eaa1311ad1ef)